libraries/testkit/syncgateway.py

```diff
@@ -95,8 +110,27 @@ def start(self, config):
             else:
                 num_replicas = get_sg_replicas(self.cluster_config)
                 playbook_vars["num_index_replicas"] = '"num_index_replicas": {},'.format(num_replicas)
+
+            if is_x509_auth(conf_path):
+                playbook_vars[
+                    "certpath"] = '"certpath": "/home/sync_gateway/certs/chain.pem",'
+                playbook_vars[
+                    "keypath"] = '"keypath": "/home/sync_gateway/certs/pkey.key",'
+                playbook_vars[
+                    "cacertpath"] = '"cacertpath": "/home/sync_gateway/certs/ca.pem",'
+                playbook_vars["server_scheme"] = "couchbases"
+                playbook_vars["server_port"] = ""
+                playbook_vars["x509_auth"] = True
+                generate_x509_certs(conf_path, bucket_names)
```

Comment:
everytime we start sync-gateway, it will generate certs or will it ignore if it is already generated and exists?

Reply:
new certs will be generated as we might switch between different cluster config and certs are generated for particular combination.